crates/matrix-sdk-ui/src/timeline/event_handler.rs

```diff
@@ -135,6 +135,8 @@ pub(super) struct TimelineEventContext {
 #[derive(Clone, Debug)]
 pub(super) enum HandleAggregationKind {
     Reaction { key: String },
+
+    Redaction,
```

Comment:
Is it possible to add a bit of documentation here please?

Also, it's really cool how everything fits together. Nice refactoring!